src/ebmc/bmc.cpp

```diff
@@ -46,6 +46,13 @@ void bmc(
     if(property.is_disabled() || property.is_failure())
       continue;
 
+    // Is it supported by the BMC engine?
+    if(!bmc_supports_property(property.expr))
+    {
+      property.failure("property not supported by BMC engine");
+      continue;
+    }
+
```

Comment:
What will be the exit code when all other properties are successfully proved? The regression tests all seem to have refuted properties.

Reply:
It should be a failure exit code, but I'll add the test for that.

Reply:
Done